### PR TITLE
PDFBox v3 upgrade plus resolution to reversed text

### PR DESCRIPTION
**Pull request by Frooodle**
# License Agreement for Contributions
By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under MPL 2.0 (Mozilla Public License Version 2.0) license. 

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: src/main/resources/templates/change-creds.html [hunk 1]
```diff
@@ -39,12 +39,12 @@ <h4></h4>
                         <h4 th:text="#{changeCreds.changeUserAndPassword}">Change Username and password</h4>
                         <form action="api/v1/user/change-username-and-password" method="post">
                             <div class="mb-3">
-                                <label for="newUsername" th:text="#{changeCreds.newUsername}">New Username</label>
-                                <input type="text" class="form-control" name="newUsername" id="newUsername" th:placeholder="${username}">
+                                <label for="username" th:text="#{changeCreds.username}">New Username</label>
```
Comment (sbplat) [inline src/main/resources/templates/change-creds.html]: Need to update .newUsername to .username in translation files.

File: src/main/resources/templates/change-creds.html [hunk 2]
```diff
@@ -39,12 +39,12 @@ <h4></h4>
...
+                                <input type="text" class="form-control" name="username" id="username" th:placeholder="${username}">
                             </div>
                             <div class="mb-3">
-                                <label for="currentPassword" th:text="#{changeCreds.oldPassword}">Old Password</label>
-                                <input type="password" class="form-control" name="currentPassword" id="currentPasswordPassword"  th:placeholder="#{changeCreds.oldPassword}">
+                                <label for="password" th:text="#{changeCreds.oldPassword}">Old Password</label>
```
Comment (sbplat) [inline src/main/resources/templates/change-creds.html:46]: Same thing here.

**Review by sbplat — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/change-creds.html (hunk 1 above)
Comment (sbplat) [reply src/main/resources/templates/change-creds.html]: Was this an intended change?

Comment (Frooodle) [reply src/main/resources/templates/change-creds.html]: It was not 
good catch 

Lazy find replacing

**Pull request merged**